src/components/SheepLogo.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+
+export const SheepLogo: React.FC = () => {
+
+    return (
```

Comment:
Does this is result SVG after SVGO optimisations?

Reply:
done